docs/channels/library_auth_reference/server-library-reference-specification.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Server library reference specification
+title: Pusher Channels | Server Library Reference Specification
```

Comment:
`Pusher Channels |` or `Pusher Channels Docs |`?

Reply:
Pusher Channels for this. Title length is important for SEO